adcm_client/objects.py

```diff
@@ -937,8 +937,19 @@ def _upload(self, bundle_stream) -> Bundle:
         return self.bundle(bundle_id=data['id'])
 
     @allure_step('Upload bundle from "{1}"')
-    def upload_from_fs(self, dirname) -> Bundle:
-        return self._upload(stream.file(dirname))
+    def upload_from_fs(self, dirname, **args) -> Bundle:
+        streams = stream.file(dirname, **args)
+        if len(streams) > 1:
+            raise TooManyArguments('Build more then one bundle from dir')
+        return self._upload(streams[0])
+
+    @allure_step('Upload bundles from "{1}"')
+    def upload_from_fs_all(self, dirname, **args) -> BundleList:
+        streams = stream.file(dirname, **args)
+        result = BundleList(self._api, empty_bundlelist='not_existing_bundle')
```

Comment:
Add comment about that dirty hack with epty_bundle_list

Reply:
Added.